example/main.js

```diff
@@ -1,12 +1,17 @@
+const { app, BrowserWindow, ipcMain } = require('electron');
+const path = require('path');
+const url = require('url');
+
+// If we're running e2e tests, use the supplied temp userData directory
+if (process.env['E2E_USERDATA_DIRECTORY']) {
+  app.setPath('userData', process.env['E2E_USERDATA_DIRECTORY']);
+}
+
```

Comment:
If the app is running e2e tests, we have to change `userData` directory very early, certainly before Sentry is started